entry.go

```diff
@@ -46,36 +52,43 @@ func (e Entry) Identity() []byte { return e.internal.Identity }
 // Note that in almost all cases, this should be the RFC6962 definition of a leaf hash.
 func (e Entry) LeafHash() []byte { return e.internal.LeafHash }
 
+// Index returns the index assigned to the entry in the log, or nil if no index has been assigned.
+func (e Entry) Index() *uint64 { return e.internal.Index }
+
+// MarshalBundleData returns this entry's data in a format ready to be appended to an EntryBundle.
```

Comment:
It does more than this, it assigns an index, right?

Reply:
Hmm, not _really_ - at least not in the sense that "(durably) assigning an index" is used elsewhere in comments, although it's true that this func is called when there's a "candidate" index for the entry. But, in here, that's only used to "render" the leaf image (and then iff this is an entry type which needs to do such a thing).